ui/tests/integration/components/license-test.js

```diff
@@ -56,7 +56,7 @@ module('Integration | Component | license info', function(hooks) {
     await render(
       hbs`<LicenseInfo @licenseId={{this.licenseId}} @expirationTime={{this.expirationTime}} @startTime={{this.startTime}} @features={{this.features}}/>`
     );
-    assert.equal(component.featureRows.length, 12, 'it renders 12 features');
+    assert.equal(component.featureRows.length, 9, 'it renders 9 features');
```

Comment:
we _could_ just have this call the helper and make sure it's the same length as the list the helper has?

Reply:
Oh yeah that's nicer

Reply:
done!